docs/DeveloperGuide.md

```diff
@@ -280,6 +281,28 @@ The sequence diagram shows how an entity command is executed:
 
 ### Command-Specific Design Considerations
 
+### Add Appointment feature
+
+**Aspect: Whether appointment fields should be optional**
+
+- **Alternative 1 (Current choice):** Sickness and medicine fields are optional.
+    - **Pros**: This allows users to create an appointment without specifying all fields initially, which is more realistic and practical as some details may not be available at the time of creation.
+
+- **Alternative 2**: Make all fields mandatory.
+    - **Pros**: Ensures complete data at the time of appointment creation, which may simplify data handling and reduce the need for future edits.
+    - **Cons**: Can be inconvenient for users who don’t have all details available immediately, possibly leading to frustration or delays in creating appointments.
+
+<br>
+
+**Aspect: What Input Should Be Valid for Fields Sickness and Medicine**
+
+- **Alternative 1 (Current choice):** Require input with at least one alphanumeric character.
```

Comment:
Should it be `at least one alphabetic character`? It's currently implemented this way